cpp/src/datetime/datetime_ops.cu

```diff
@@ -54,6 +54,8 @@ enum class datetime_component {
   NANOSECOND
 };
 
+enum class rounding_kind { CEIL, FLOOR, ROUND };
```

Comment:
Remove `ROUND` until we expose APIs for that in a follow-up PR.
```suggestion
enum class rounding_kind { CEIL, FLOOR };
```

Reply:
Resolved by 052456a